hack/e2e-kind-cluster-setup.sh

```diff
@@ -25,18 +29,42 @@ setup_cluster() {
 push_local_image() {
     OCI_BIN="$OCI_BIN" IMAGE_REGISTRY="$IMG_REGISTRY" IMAGE_TAG="$IMG_TAG" make manifests
     OCI_BIN="$OCI_BIN" IMAGE_REGISTRY="$IMG_REGISTRY" IMAGE_TAG="$IMG_TAG" make img-build
-    kind load docker-image $IMG_REGISTRY/multus-dynamic-networks-controller:$IMG_TAG
+    if [[ $KUBEVIRT_PROVIDER != "external" ]]; then
+      kind load docker-image $IMG_REGISTRY/multus-dynamic-networks-controller:$IMG_TAG
+    else
+      "$OCI_BIN" push $IMG_REGISTRY/multus-dynamic-networks-controller:$IMG_TAG 
+    fi
 }
 
 cleanup() {
     rm -rf multus-cni
     git checkout -- manifests/
 }
 
+install_multus() {
+  local version="v4.1.1"
```

Comment:
now sure how I feel about having to track multus releases ... 

At a glance, I would rather have something else.

Reply:
I can have a variable VERSION, with default value "latest" that does as now
and if a version is exported then it will deploy the version
this way it will be version agnostic

Reply:
Good enough for me.